### ISSUE TITLE
Add note about minimum server memory for SQL Servers

### ISSUE THREAD

**Issue by Stroell** [MicrosoftDocs/memdocs#570]
In the _General sizing guidelines_ should be a note/warning that you should not configure SQL Server minimum and maximum server memory to the same value. 

The stated _SQL memory allocation_ in the table can be a little bit misleading. It is important that you configure a reasonable value for the minimum server memory, so the OS can “breathe” in case of memory pressure. This is extremely important when the Primary Site and SQL run on the same box but applies to dedicated SQL Servers as well. There should be some sort of guidance which value maces sense. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cda5d744-c604-5cf1-2174-44a82db01053
* Version Independent ID: fd18ea2a-485e-67b0-dcbf-efa9784e2d3e
* Content: [Site size and performance guidelines - Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/configs/site-size-performance-guidelines)
* Content Source: [memdocs/configmgr/core/plan-design/configs/site-size-performance-guidelines.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/configmgr/core/plan-design/configs/site-size-performance-guidelines.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @Stroell for the feedback.

1. This first point seems obvious to me, but I suppose it doesn't hurt to explicitly state it.

2. Can you clarify further on the second point? What you describe is the point of that column 😄 Don't set SQL to use more than 65% to leave room for the OS or co-located site roles. But I suppose we could add a note to more explicitly state that.

Do these comments align with the original intent of your feedback?

**Comment by Stroell** [comment]
Hi Aaron, thanks for your quick response!
Actually it´s only one request and not two separate points. Maybe it was not a good wording from my end. Let me try again.

The table _General sizing guidelines_ includes the column "SQL memory allocation" I guess this means the _Maximum server memory_ configuration in SQL 
![grafik](https://user-images.githubusercontent.com/42348774/89529373-e86e0700-d7ec-11ea-8393-67f59c75e7d0.png)

However this can be misunderstood (happened at my customer) to configure _Minimum servery memory_ and _Maximum server memory_ to the same value - which is a bad configuration.

Therefore there should be:

- A note/warning that you should not configure min and max to the same value

- or recommended values for the min/max configuration

**Comment by aczechowski** [comment]
I submitted a fix for this, adding a new note below the table. This issue will auto close when my change merges. It will go live with the next daily publishing process, which should be later today.
